test/lib/validation/schema/specVersion/2.0/kind/project/application.js

```diff
@@ -630,3 +630,24 @@ test("framework configuration: Invalid", async (t) => {
 		}
 	]);
 });
+
+test("framework configuration: Missing 'name'", async (t) => {
+	await assertValidation(t, {
+		"specVersion": "2.0",
+		"type": "application",
+		"metadata": {
+			"name": "my-application"
+		},
+		"framework": {}
+	}, [
+		{
+			dataPath: "/framework",
+			keyword: "required",
+			message: "should have required property 'name'",
```

Comment:
must have required property 'name'

Reply:
That's the message coming from the "ajv" library. It's just here as we compare the whole error objects returned by ajv. But we won't use it to create our messages.